azure_functions_worker/bindings/meta.py

```diff
@@ -25,15 +34,43 @@ def load_binding_registry() -> None:
     global BINDING_REGISTRY
     BINDING_REGISTRY = func.get_binding_registry()
 
-
-def get_binding(bind_name: str) -> object:
+    if BINDING_REGISTRY is None:
+        # If the BINDING_REGISTRY is None, azure-functions hasn't been
+        # loaded in properly.
+        raise AttributeError('BINDING_REGISTRY is None. Sys Path:'
```

Comment:
Lets improve the error - if it shows up to the customer, it should have soem more meaning.

You can mention - BINDING_REGISTRY is none, and is because the worker couldn't find the `azure-function` library.

Reply:
Added info about library not being found